account_invoice_blocking/models/account_invoice.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Acsone SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class AccountInvoice(models.Model):
+
+    _inherit = 'account.invoice'
+
+    @api.model
+    def _get_move_line(self, invoice):
```

Comment:
Please convert this method from `@api.model` to `@api.multi` + `self.ensure_one()` without the *self recordset* argument, to follow standard changes since v10.